modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/Carousel.es.js

```diff
@@ -91,29 +91,38 @@ const Carousel = ({
 	handleClickNext,
 	handleClickPrevious,
 	showArrows = true,
-}) => (
-	<div className="carousel closed sidenav-container">
-		<InfoPanel metadata={currentItem.metadata} />
-
-		<div className="sidenav-content">
-			{showArrows && (
-				<Arrow direction="left" handleClick={handleClickPrevious} />
-			)}
-
-			{currentItem.returntype === STR_VIDEO_HTML_RETURN_TYPE ? (
-				<PreviewVideo html={currentItem.value} />
-			) : (
-				<PreviewImage
-					src={currentItem.url || currentItem.base64}
-					title={currentItem.title}
-				/>
-			)}
-
-			{showArrows && (
-				<Arrow direction="right" handleClick={handleClickNext} />
-			)}
+}) => {
+	const isVideo = currentItem.returntype === STR_VIDEO_HTML_RETURN_TYPE;
```

Comment:
maybe we will have to change this in the future for the shortcut item selector?

![](https://media2.giphy.com/media/ZZkCo8zKWtt2ZgozfX/giphy.gif)